external/metabolism/modifiers.lua

```diff
@@ -33,9 +36,16 @@ modifiers = function(user, data)
 		nick = nick.."-chan"
 	end
 	local finalmsg = "<"..nick..">"..msg
-	if isthere(user.sNick, falone) then
-		Core.SendToUser(user, finalmsg)
+	if isthere( user.sNick, falone ) then
+		Core.SendToUser( user, finalmsg )
 		return false
 	end
 	return finalmsg
 end
+
+function GarbleMessage( sLine )
```

Comment:
Since all the keys in `tGarble` are single characters; you can replace the whole function to be:

``` Lua
return sLine:gsub( '.', tGarble )
```